src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
+ReactDOM.render(
   <React.StrictMode>
     <App />
```

Comment:
- [ ] kindly keep to the design guide
 your design :point_down: 
   
![Screenshot from 2022-07-06 10-57-55](https://user-images.githubusercontent.com/66097371/177524952-6df3922a-a251-4d25-bcc7-13d92df2ba1b.png)
design guide :point_down: 
![Screenshot from 2022-07-06 10-59-34](https://user-images.githubusercontent.com/66097371/177525140-b7334906-19e1-4f64-83f0-bd02bd3f02b6.png)